test/integration/visitor_can_edit_cart_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class VisitorCanEditCartTest < ActionDispatch::IntegrationTest
+  test "visitor can remove an item from his or her cart" do
+    category_1 = Oil.create(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_id: category_2.id)
+    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_id: category_2.id)
+    item_4 = Chip.create(name: "Lard Yummies", price: 19, description: "Chock Full of Lard", oil_id: category_1.id)
```

Comment:
Useless assignment to variable - `item_4`. Did you mean `item_1`?<br>Line is too long. [115/80]